bundles/org.openhab.binding.shelly/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -341,12 +347,29 @@
 		<kind>trigger</kind>
 		<label>Event</label>
 		<description>Event Trigger, e.g. button on/off, Output on/off</description>
-		<event />
+		<event>
+			<options>
+				<option value="ROLLER_OPEN">sensor data updated</option>
```

Comment:
You should use different human readable label texts for the options, e.g. "Roller open".

Reply:
done